docs/coordinates/satellites.rst

```diff
@@ -0,0 +1,84 @@
+.. include:: references.txt
+
+.. _astropy-coordinates-satellites:
+
+Working with Earth satellites using Astropy Coordinates
+*******************************************************
+
+Finding ``TEME`` coordinates from Two-Line-Ephemerides (TLE)
+============================================================
+
+There is currently no support in `astropy.coordinates` for computing satellite orbits
+from the TLE orbital elements avaiable from a source like `Celestrak <http://celestrak.com/>`_.
+Full support for handling TLE files is available in the `Skyfield <http://rhodesmill.org/skyfield/>`_
+library, but if you do not wish to install that library, advice for dealing with satellite data in
+astropy is below.
+
+..
+  EXAMPLE START
+  Using spg4 to get a TEME coordinate
+
+You will need some external library to compute the position and velocity of the satellite from the
+TLE orbital elements. The `spgp4 <https://pypi.org/project/sgp4/>`_ library can do this. An example
+using this library to find the  `~astropy.coordinates.TEME` coordinates of a satellite is::
+
+        >>> from sgp4.api import Satrec
```

Comment:
I think you may need some `doctest requires sgp4` or so for this to work. Which also means `sgp4` should be included in the depedencies...

Reply:
have attempted this now.